### PR TITLE
support for JDK 1.8, maven 3.3.9, node 4.4.7, seed job and all plugins

### LINKED ISSUE #3: Figure Out JDK Issues
Base image doesn't have JDK, how do we handle this?
```
[Pipeline] sh
[workspace] Running shell script
+ java -version
openjdk version "1.8.0_91"
OpenJDK Runtime Environment (build 1.8.0_91-b14)
OpenJDK 64-Bit Server VM (build 25.91-b14, mixed mode)
[Pipeline] sh
[workspace] Running shell script
+ javac -version
/var/lib/jenkins/jobs/a/workspace/.jenkins-5e2ab0e4/script.sh: line 2: javac: command not found
```

### LINKED ISSUE #4: Make Sure JAVA_HOME Is Set For mvn
```
[Pipeline] tool
Unpacking https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.3.9/apache-maven-3.3.9-bin.zip to /var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/mvn-3 on Jenkins
[Pipeline] sh
[workspace] Running shell script
+ /var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/mvn-3/bin/mvn --version
/var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/mvn-3/bin/mvn: line 143: which: command not found
/var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/mvn-3/bin/mvn: line 171: which: command not found
Error: JAVA_HOME is not defined correctly.
  We cannot execute 

```

### PR DESCRIPTION
**Pull request by sherl0cks**
Initial build out. 
#### Which tests illustrate how this code works?
At the moment, this has only been testing manually. This is going to be quite hard to test in CI...

#### Is there a relevant Issue open for this?
resolves #4 
resolves #3 

#### Are there any other relevant resources that should be reviewed as well?
See README.md

#### Who would you like to review this?
/cc @oybed @mdanter @mcanoy

### CODE REVIEW

File: configuration/jobs/release-pipeline-seed-job/config.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            <name/>
+            <resourceVersion/>
+            <contextDir/>
+        </io.fabric8.jenkins.openshiftsync.BuildConfigProjectProperty>
+        <hudson.model.ParametersDefinitionProperty>
+            <parameterDefinitions>
+                <hudson.model.StringParameterDefinition>
+                    <name>git_url</name>
+                    <description/>
+                    <defaultValue>https://github.com/sherl0cks/infographic.git</defaultValue>
```
Comment (mcanoy) [inline configuration/jobs/release-pipeline-seed-job/config.xml]: Probably don't want a default value here. Future removal should be fine

File: helpers/writePluginFile.groovy [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env groovy
+
+// This script doesn't parse cli args right now, you'll need to modify it directly
+
+String user = 'admin'
```
Comment (mcanoy) [inline helpers/writePluginFile.groovy]: Not 100% what is going on here. I am assuming these hard coded values will change in future revisions. True? Does jenkins need to be deployed at apiHost=localhost:8080 to work?

Comment (sherl0cks) [inline helpers/writePluginFile.groovy]: This is a helper script I wrote to extract the plugin list from an existing Jenkins build. This is not production code, just a tool to help dev. If you want to use it, modify the script directly. More on this in the README.

File: configuration/jobs/release-pipeline-seed-job/config.xml (hunk 1 above)
Comment (sherl0cks) [inline configuration/jobs/release-pipeline-seed-job/config.xml]: yea we can look at this later: https://github.com/rht-labs/openshift-jenkins-s2i-config/issues/6

File: configuration/config.xml.tpl [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<hudson>
+  <disabledAdministrativeMonitors/>
+  <version>1.651.2</version>
```
Comment (oybed) [inline configuration/config.xml.tpl]: It scares me keeping full config files like this due to change in versions as we move forward to newer versions. Yes, it should be backwards compatible, but still...

Comment (sherl0cks) [inline configuration/config.xml.tpl]: @oybed honestly, I'm copy and pasting these right out of `JENKINS_HOME.` Moving to new versions is going to be a similar process. I learned how to do this from scratch in 24 hours, so I'm not really concerned on maintenance. However, we should probably be a bit more diligent about when/why we upgrade after this new round of changes is made.

1. Config new host
2. Pull file iteratively
3. Build image

**Comment by sherl0cks** [comment]
@oybed we'll need cut release branches as that point, and then have build processes aligned to the distinct releases. most likely core infra stuff stays on master and engagement stuff is broken out as release branches

**Pull request merged**